ddtrace/gitmetadata.py

```diff
@@ -0,0 +1,88 @@
+import os
+import inspect
+import subprocess
+import pkg_resources
```

Comment:
we should avoid importing `pkg_resources`. by default when imported it builds a working set of all packages, so it iterates all packages and parses their metadata. it is really really really slow. the overhead it adds to startup makes it basically a non-starter for serverless environments.